webapp/src/main/java/com/box/l10n/mojito/entity/ScheduledJobStatusEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.box.l10n.mojito.entity;
+
+import static org.hibernate.envers.RelationTargetAuditMode.NOT_AUDITED;
+
+import com.box.l10n.mojito.rest.View;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobStatus;
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.persistence.Basic;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.hibernate.envers.Audited;
+
+@Entity
+@Table(name = "scheduled_job_status_type")
+@Audited(targetAuditMode = NOT_AUDITED)
```

Comment:
Out of curiosity, is this needed? I would assume that the lack of annotation likely means that it's not audited.

Reply:
This is needed, it will try place a scheduled_job_status_type row into an audited table as it is used in a JOIN on the scheduled_job table, it was causing me issues as it kept attempting to insert into an audited table that didn't exist for the job status.